packages/payment/src/migrations/Migration20240225134525.ts

```diff
@@ -28,6 +28,7 @@ export class Migration20240225134525 extends Migration {
         ALTER TABLE IF EXISTS "payment_session" ADD COLUMN IF NOT EXISTS "payment_authorized_at" TIMESTAMPTZ NULL;
         ALTER TABLE IF EXISTS "payment_session" ADD COLUMN IF NOT EXISTS "raw_amount" JSONB NOT NULL;
         ALTER TABLE IF EXISTS "payment_session" ADD COLUMN IF NOT EXISTS "deleted_at" TIMESTAMPTZ NULL;
+        ALTER TABLE IF EXISTS "payment_session" ADD COLUMN IF NOT EXISTS "context" JSONB NULL;
```

Comment:
suggestion: I believe the payment module does not have the appropriate config for the dev mikro config and is outdated in terms of loaders etc, should we take care of that, not necessecarly here